lib/reporters/mini.js

```diff
@@ -11,9 +11,9 @@ var repeating = require('repeating');
 var colors = require('../colors');
 
 chalk.enabled = true;
-Object.keys(colors).forEach(function (key) {
+for (let key of Object.keys(colors)) {
 	colors[key].enabled = true;
-});
+}
```

Comment:
@Qix- This is needed because we assign shortcuts to the colors in [`lib/colors.js`](https://github.com/avajs/ava/blob/9616dde582e5333c3528cdd217c37e5b41798c9a/lib/colors.js) (context https://github.com/avajs/ava/pull/1104#issue-187486491) Any way we could make the above loop not nessecary? Maybe if we made `.enabled` property on the prototype?